README.md

```diff
@@ -39,25 +40,13 @@ module.exports = {
 +    project: "./tsconfig.json",
 +    ecmaVersion: 2018,
 +    sourceType: "module"
++    tsconfigRootDir: __dirname,
```

Comment:
This helps for mono repo setups where you have multiple projects with different ESLint configs, but can't be part of the base config.